initialize.go

```diff
@@ -8,9 +8,22 @@ import (
 )
 
 type Config struct {
-	Stats globalstats.Config
+	// Stats specifies the config for reporting stats to the global
+	// ctlstore stats namespace.
+	//
+	// By default, global stats are enabled with a set of sane defaults.
+	Stats *globalstats.Config
+
+	// LDBVersioning, if enabled, will instruct ctlstore to look for
+	// LDBs inside of timestamp-delimited folders, and ctlstore will
+	// hot-reload new LDBs as they appear.
+	//
+	// By default, this is disabled.
+	LDBVersioning *bool
```

Comment:
The zero value for a bool is `false` so I think it makes sense to just use that instead of a `*bool`.